tests/integration_tests_rust.rs

```diff
@@ -995,7 +995,7 @@ async fn splice_channel() {
 	// Splice-in funds for Node B so that it has outbound liquidity to make a payment
 	node_b.splice_in(&user_channel_id_b, node_a.node_id(), 4_000_000).unwrap();
 
-	expect_splice_pending_event!(node_a, node_b.node_id());
+	let txo = expect_splice_pending_event!(node_a, node_b.node_id());
```

Comment:
These test changes pass on `main`. Can you add coverage for the cases where the previous approach would lead to inaccurate fee estimations?